config/samples/apm/apm_es_kibana.yaml

```diff
@@ -9,6 +9,8 @@ spec:
   nodeSets:
   - name: default
     count: 3
+    config:
+      node.store.allow_mmap: false
```

Comment:
Would be nice to include a comment that points to the relevant doc section about how this may be bad for performance: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-virtual-memory.html